src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -82,4 +100,36 @@ public Command parseCommand(String userInput) throws ParseException {
         }
     }
 
+    /**
+     * Converts an input command into a full command if input command is a prefix.
+     *
+     * @param command user input command string
+     * @return full command if input is a prefix, else return the original input command
+     * @throws ParseException if the input matches to multiple commands
```

Comment:
I think "matches multiple commands" is grammatically correct.

Also we use full stop after each line, and capitalization at the start of each line as appropriate.

https://se-education.org/guides/conventions/java/intermediate.html#comments